src/model/FloorspaceReverseTranslator.cpp

```diff
@@ -602,34 +582,43 @@ namespace model {
 
           // Create a window subsurface for every window that is on this edge
           for (const auto& window : m_currentFSStory->windows()) {
-            if (window.edge()->id() == edgeRef.edge().id()) {
-              createWindowSubsurface(window, surface, edgeRef, minZ, maxZ);
+            // Get window's x/y position which is the alpha based on its edge
+            // windows can have multiple alphas but we only need one
+            auto edge = *window.edge();
```

Comment:
We're sure it's inintialized? (I realize this was existing code, the -> operator was called already before)

Reply:
Yes confident about that otherwise the floorspace model would be invalid but lets make 100% sure